core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/settings/SettingsViewModel.kt

```diff
@@ -39,11 +40,14 @@ data class SettingsOfflineStorageOption(
 )
 
 data class SettingsUiState(
-
+    val sendAppOptionVisible: Boolean = false,
```

Comment:
Properties related to backup etc. are not related to this pull request, should not be here. sendAppOptionVisible is no longer used here (now it is moved to accountlist), so should also be removed from this uiState.